python/llm/example/GPU/QLoRA-FineTuning/qlora_finetuning.py

```diff
@@ -62,6 +62,7 @@
     )
     model = get_peft_model(model, config)
     cast_lora_weight(model, torch.bfloat16)
+    torch.xpu.synchronize()
```

Comment:
Maybe we should move these two into `get_peft_model` (if it's xpu, we simply cast the weight and synchronize)